src/main/java/org/isf/admission/service/AdmissionIoOperations.java

```diff
@@ -112,7 +116,7 @@ public List<AdmittedPatient> getAdmittedPatients(String searchTerms) throws OHSe
 	 */
 	public List<AdmittedPatient> getAdmittedPatients(String searchTerms, LocalDateTime[] admissionRange, LocalDateTime[] dischargeRange)
 			throws OHServiceException {
-		return repository.findPatientAdmissionsBySearchAndDateRanges(searchTerms, admissionRange, dischargeRange);
+		return repositoryImp.findPatientAdmissionsBySearchAndDateRanges(searchTerms, admissionRange, dischargeRange);
```

Comment:
why was the change to `repositoryImp` when the same change wasn't made in line 105 above?   They are both calling the same method aren't they?

Reply:
because `repository ` extends `AdmissionIoOperationRepositoryCustom` that is an interface that is implemented by `repositoryImp`, so both calls are valid. Anyway for the same reason we don't need to autowire `*Impl` classes, roll back like in 105